.devcontainer/Dockerfile

```
@@ -0,0 +1,34 @@
+FROM node:12-bullseye-slim
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get -y install --no-install-recommends \
+        locales build-essential git curl sudo python3-dev python3-venv python3-pip ca-certificates unzip openssh-client \
+        default-jdk libpq-dev postgresql postgresql-client graphviz libmagic-dev ruby-dev ruby-rubygems nginx-light redis python2 \
+        libjpeg-dev libtiff-dev libwebp-dev liblcms2-dev \
```

I'd prefer to see on package per line, alphabetically sorted, so it's easier to review later if dependencies are changed:
```suggestion
    && apt-get -y install --no-install-recommends \
        build-essential \
        ca-certificates \
        curl \
        default-jdk \
        git \
        graphviz \
        libjpeg-dev \
        liblcms2-dev \
        libmagic-dev \
        libpq-dev \
        libtiff-dev \
        libwebp-dev \
        locales \
        nginx-light \
        openssh-client \
        postgresql \
        postgresql-client \
        python2 \
        python3-dev \
        python3-pip \
        python3-venv \
        redis \
        ruby-dev \
        ruby-rubygems \
        sudo \
        unzip \
```